RELEASE.md

```diff
@@ -0,0 +1,93 @@
+<!--
+# Copyright 2018-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Release Notes for 2.23.0
+
+## New Freatures and Improvements
+
+* Auto-generated model configuration enables 
+  [dynamic batching](https://github.com/triton-inference-server/server/blob/r22.06/docs/model_configuration.md#default-max-batch-size-and-dynamic-batcher) 
+  in supported models by default.
+
+* Python backend models now support 
+  [auto-generated model configuration](https://github.com/triton-inference-server/python_backend/tree/r22.06#auto_complete_config).
+
+* [Decoupled API](https://github.com/triton-inference-server/server/blob/r22.06/docs/decoupled_models.md#python-model-using-python-backend) 
+  support in Python Backend model is out of beta.
+
+* Updated I/O tensors 
+  [naming convention](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#special-conventions-for-pytorch-backend) 
+  for serving TorchScript models via PyTorch backend.
```

Comment:
Should also add =>
"Improvements to Perf Analyzer stability and profiling logic."

Reply:
I have added missed statement to the documentation, thanks for the catch